File: app/src/main/java/nic/goi/aarogyasetu/models/network/FCMTokenObject.java
package nic.goi.aarogyasetu.models.network;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class FCMTokenObject{

    @SerializedName("ft")
    @Expose
    private String ft;
    public FCMTokenObject(String ft){
        this.ft = ft;
    }

}

File: app/src/main/java/nic/goi/aarogyasetu/models/network/FCMTokenObject.kt
package nic.goi.aarogyasetu.models.network

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class FCMTokenObject(
    @field:SerializedName("ft")
    @field:Expose
    private val ft: String
)
